README.md

```diff
@@ -129,9 +129,14 @@ strategy:
 ```
 
 ## Setting up CLion 
+
+### Opening project
 Configuring CLion with the extension template requires a little work. Firstly, make sure that the DuckDB submodule is available. 
 Then make sure to open `./duckdb/CMakeLists.txt` (so not the top level `CMakeLists.txt` file from this repo) as a project in CLion.
 Now to fix your project path go to `tools->CMake->Change Project Root`([docs](https://www.jetbrains.com/help/clion/change-project-root-directory.html)) to set the project root to the root dir of this repo.
 
-Now to configure the build targets, copy the CMake variables specified in the Makefile and ensure
-the build directory is set to `../build/<build_mode>`.
\ No newline at end of file
+
+### Debugging
+To set up debugging in CLion, there are two simple steps required. Firstly, in `CLion -> Settings -> Build, Execution, Deploy -> CMake` you will need to add the desired builds (e.g. Debug, Release, RelDebug, etc). There's different ways to configure this, but the easiest is to leave all empty, except the `build path`, which needs to be set to `../build/{build type}`. Now on a clean repository you will first need to run `make {build type}` to initialize the CMake build directory. After running make, you will be able to (re)build from CLion by using the build target we just created.
```

Comment:
When i execute the configured target and execute it, the source is compiled and linked and the executable is started but i didn't see the prompt


```
/Users/jima/stoic/workspace/pdb-duckdb-ext/duckdb/build/debug/duckdb
duckdb(88310,0x21b0b4600) malloc: nano zone abandoned due to inability to preallocate reserved vm space.
v0.7.0 f7827396d7
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
```

Reply:
i never see the D

Reply:
![CleanShot 2023-03-01 at 12 40 26@2x](https://user-images.githubusercontent.com/1454089/222023034-22fbdfe2-6e85-46e5-9d86-33bda0fb61bf.png)
<img width="991" alt="CleanShot 2023-03-01 at 12 42 07@2x" src="https://user-images.githubusercontent.com/1454089/222023084-2e73281e-f1f6-4521-9c83-5e0efbc61ce8.png">

Reply:
DuckDB CLI is a bit flaky when ran from within clion. it is recommended to use the unittest binary as a target. Running the duckdb cli works fine from your clion terminal though! Added a comment to warn users for this!